### PR TITLE
feat: to only include one dicom file name for each subject in the email

### PR DESCRIPTION
**Pull request by kcho**
The email becomes very laggy when all the dicom file names are included. This PR updates the second part of the table in the email, to only include one dicom file name for each subject.

### CODE REVIEW

File: lochness/email/__init__.py [hunk 1]
```diff
@@ -156,6 +156,21 @@ def send_out_daily_updates(Lochness, days: int = 1, test: bool = False):
...
                                   'Datatype', 'Download time (UTC)']
         s3_df_selected.reset_index().drop('index', axis=1, inplace=True)
+
+        # too many dicom file names -> remove
+        subject_mri_gb = s3_df_selected[
+                s3_df_selected.Datatype == 'mri'].groupby('subject')
+
+        sample_mri_df = pd.DataFrame()
+        for _, table in subject_mri_gb:
+            subject_mri_sample = table.iloc[:2]  # select only two raws
```
Comment (tashrifbillah) [inline lochness/email/__init__.py:166]: Is this the shorthand for `subject_mri_sample.iloc[0]`?

File: lochness/email/__init__.py [hunk 2]
```diff
@@ -156,6 +156,21 @@ def send_out_daily_updates(Lochness, days: int = 1, test: bool = False):
...
+            subject_mri_sample.iloc[1]['File name'] = '...'
+            sample_mri_df = pd.concat([sample_mri_df, subject_mri_sample])
+
+        s3_df_selected = pd.concat([
+            s3_df_selected[s3_df_selected.Datatype != 'mri'],
+            sample_mri_df]).sort_index()
```
Comment (tashrifbillah) [inline lochness/email/__init__.py:172]: After `group_by()`, I have only used `reset_index()`. Can you tell briefly why it is `sort_index()`?

**Review by tashrifbillah — APPROVED** [review]
This is a simple yet elegant design. Tashrif has asked two questions. Upon answering/testing properly, Kevin can merge.

File: lochness/email/__init__.py (hunk 1 above)
Comment (kcho) [reply lochness/email/__init__.py:166]: no, `.iloc[0]` returns the first row in `series` format, `.iloc[:2]` returns the first two rows in `dataframe` format.

File: lochness/email/__init__.py (hunk 2 above)
Comment (kcho) [reply lochness/email/__init__.py:172]: I'm adding `.sort_index()` for the final dataframe have the original order to rows. Without this, all the mri rows will be attached at the end of the table, which I do not want.

File: lochness/email/__init__.py (hunk 1 above)
Comment (tashrifbillah) [reply lochness/email/__init__.py:166]: Sorry, that's not what I asked. In the next line, you are assigning a value to `subject_mri_sample.iloc[1]` which is 2nd row. What row are you assigning by `subject_mri_sample` only?

Comment (kcho) [reply lochness/email/__init__.py:166]: oh I'm trying to keep one dicom file name followed by '...'., for each subject. Does it answer your question?

Comment (tashrifbillah) [reply lochness/email/__init__.py:166]: Tashrif is answering his question:

* `subject_mri_sample` data frame has only two rows. This variable was not declared before. It is declared and assigned in one step as a data frame from right hand side.
* `subject_mri_sample.iloc[1]['File name'] = '...'` overwrites the `File name` in 2nd row with ellipsis

**Pull request merged**